config/webpack/demo/webpack.config.dev.js

```diff
@@ -12,6 +12,7 @@ var FILES = PACKAGES.map(function (p) {
 var DEMO = path.resolve("demo");
 var WDS_PORT = 3000;
 
+// TODO: Can we combine all three webpack configs???
```

Comment:
Since these configs are just for the demo project, I'm not sure I understand either why there are multiple configs.